en-US/Migration_Guide/5e_Migration_Guide.md

```diff
@@ -42,6 +48,48 @@ If a product (e.g. an adventure or a monster stat block) mentions one of these s
 - Telepathic Bond → Rantock's Telepathic Bond
 - Tiny Hut → Melestrua's Marvelous Marquee
 
-<!-- Not yet renamed:
-- Resilient Sphere → ???
--->
\ No newline at end of file
+<!-- Not renamed: Resilient Sphere -->
+
+### Reworded spells
+
+Many spells were reworded for clarity and/or brevity, but remain the same mechanically.
+
+### Changes related to pinpoint
+
+As explained in the section on [Pinpoint](#pinpoint), many spells in the SRD 5.1 use language such as "a creature you can see" or "a creature that can hear you".
+Since Free5e expands upon the senses that can be used in many places, spells that used such language also had to be updated.
+
+### More expansive changes to spells
+
+In addition to the System Reference Document 5.1 by Wizards of the Coast LLC, Free5e also takes inspiration and uses content from the A5E System Reference Document (A5ESRD) by EN Publishing (available at [A5ESRD.com](https://A5ESRD.com) under the Creative Commons Attribution 4.0 International License, [https://creativecommons.org/licenses/by/4.0/legalcode](https://creativecommons.org/licenses/by/4.0/legalcode)).
+Specifically, many spells were modified or expanded upon by in the A5ESRD.
+Free5e uses many of the updated spells, though always keeping compatibility with published adventures in mind.
+
+In some cases, using the A5ESRD version of a spell would have broken such compatibility.
+In most such cases, Free5e either adjusted the spell to enhance compatibility, or reverted the spell to the SRD 5.1 version.
+
+<!-- TODO It would be helpful to go into more detail on each changed spell here; however that's also A LOT of work. Maybe we can add those at a later point in time. -->
+
+### Additional spells
+
+Free5e contains a number of spells that are not included in the SRD 5.1.
+Specifically, the following spells were added:
+
+- Ceremony
+- Charm Monster
+- Friends
+- Hex
+- Iz’zart's Swarm Limb
+
+While you may be familiar with other spells of those names, the ones included in Free5e are either taken (and possibly adapted) from the A5ESRD or they were written specifically for Free5e.
+
+## Senses
+
+### Pinpoint
+
+Check the article [This D&D Rule Change Finally Makes Sense](https://wyrmworkspublishing.com/this-dd-rule-change-finally-makes-sense/) for details
```

Comment:
Note: I would like to copy the full article into this document. However since I have a few questions before doing so, I won't include it as part of this pull request.

The same is true for the article about Blindsight and Parasense.